doc/configuration_options.rst

```diff
@@ -446,6 +446,11 @@ The following options are valid in version ``1`` (beside the generic options):
   checkout into the workspace with their branch specified in the ``source``
   entry.
 
+* ``show_results_images``: a list of workspace-relative paths to images
```

Comment:
Since the images could contain anything beside performance test results how about naming the option `show_images` instead?